api/src/main/java/io/opentelemetry/trace/propagation/HttpTraceContext.java

```diff
@@ -198,6 +199,11 @@ private static SpanContext extractContextFromTraceParent(String traceparent) {
     }
 
     try {
+      String version = traceparent.substring(0, 2);
```

Comment:
Shouldn't we check whether there is a trailing `-` too? What does W3C say?

Reply:
it's already checked above:
```
traceparent.charAt(TRACE_ID_OFFSET - 1) == TRACEPARENT_DELIMITER
```